Packs/RsaNetWitnessPacketsAndLogs/Integrations/integration-RsaNetWitnessPacketsAndLogs_README.md

```diff
@@ -1477,4 +1477,7 @@
 </tr>
 </tbody>
 </table>
-<p> </p>
\ No newline at end of file
+<p> </p>
+<hr>
+<h2>Troubleshooting</h2>
+<p>When executing the <code>!netwitness-query</code> command and you wish to use the <code>sessionid</code> rsa netwitness meta key as part of the query passed to the <code>query</code> argument, please note that for any given session that Decoders, Concentrators, and Brokers appliances all maintain their own Session IDs which may be different. You can read more about using Session IDs <a href="https://community.rsa.com/docs/DOC-67073">here</a>.</p>
```

Comment:
```suggestion
<p>When executing the <code>!netwitness-query</code> command and you wish to use the <code>sessionid</code> RSA Netwitness meta key as part of the query passed to the <code>query</code> argument, please note that for any given session, that Decoders, Concentrators, and Brokers appliances all maintain their own Session IDs, may be different. You can read more about using Session IDs <a href="https://community.rsa.com/docs/DOC-67073">here</a>.</p>
```

Reply:
You're right about needing to change `rsa netwitness` to `RSA NetWitness` but I don't think you are correct about the commas. I did find a different mistake I made though.